getting_started.md

```diff
@@ -0,0 +1,62 @@
+# Building, Generating, and Uploading to S3
+## Installation Requirements
+### MATLAB   
+1. MATLAB                    - Required  
+2. Simulink                  - Required  
+3. HDL Coder                 - Required  
+4. DSP System Toolbox        - Strongly Suggested, Required for Example Designs  
+5. Fixed-Point Designer      - Strongly Suggested, Required for Example Designs  
+6. Signal Processing Toolbox - Strongly Suggested, Required for Example Designs  
+7. HDL Verifier              - Strongly Suggested  
+8. MATLAB Coder              - Strongly Suggested     
+9. Simulink Coder            - Strongly Suggested  
+
+### Quartus  
+1. Standard - Arria 10 for the Audio Blade and Audio Research, Cyclone V for the Audio Mini  
+or 
+2. Lite     - Cyclone V for the Audio Mini
+
+### Windows Subsystem for Linux: Ubuntu 18.04 LTS or newer 
+#### Run the following install commands 
+1. `sudo apt update` 
+2. `sudo apt install gcc-arm-linux-gnueabihf -y`  
+3. `sudo apt install device-tree-compiler -y` 
+4. `sudo apt install wsl -y`
+5. `sudo apt install dos2unix -y` 
+6. `sudo apt install libncurses-dev -y`
+7. `sudo apt install libncurses-dev -y`
+8. `sudo apt install libssl-dev -y`
+
+#### Setting Up the Linux SoC-FPGA on WSL
+1. `cd ~; git clone https://github.com/fpga-open-speech-tools/linux-socfpga.git`
+2. `cd linux-socfpga`
+3. `cp arch/arm/configs/openspeech_defconfig .config`
+4. `make menu config ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf-` - Save and Exit
```

Comment:
This functionality can be replaced with `make openspeech_defconfig`

Reply:
I attempted to replace the two lines above with `make openspeech_defconfig;` in `FrOST_Autogen_Config.sh` but got the following error
![linux-socfpga-error](https://user-images.githubusercontent.com/53094583/94457846-5e01ae00-0172-11eb-83fe-3967107aeb5b.PNG)

Reply:
Reverted to copying the config and running make menuconfig - Commit 4f63467

Reply:
Oops, we need to export the cross compile environment variables. So either 
```bash
export ARCH=arm
export CROSS_COMPILE=arm-linux-gnueabihf
make openspeech_defconfig
```
or
```bash
make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf openspeech_defconfig
```

Both of those should hopefully work. In your screenshot, it was trying to pull the config file from arch/x86/configs, but it exists in arch/arm/configs.

Reply:
Updated with Commit 29f0c4c